pkg/platform/packet/packet_test.go

```diff
@@ -355,3 +355,86 @@ func TestTerraformAddDeps(t *testing.T) {
 		})
 	}
 }
+
+func Test_findRepeatingString(t *testing.T) {
```

Comment:
IMO this is an implementation detail, we don't need to test it. Just testing the configuration on high level is enough.

Reply:
I think testing things at the function level using unit test is a good strategy to write readable code.

Reply:
I don't think a test hurts.